### ISSUE TITLE
Sign up screen for viewer invite wording says "start editing"

### ISSUE THREAD

**Issue by jartes** [Automattic/wp-calypso#95171]
### Quick summary

When inviting a user to an Atomic or Simple site, when following the confirmation link sent via email, and the invited user is not logged in on WordPres.com, the signup screen says "Sign up to start editing {site-title}"

### Steps to reproduce

1. On a Simple or Atomic private site, invite a user as a viewer. 
2. Click on the confirmation link received via email and make sure you are not logged into WordPress.com.
3. Check the sign-up screen text.

### What you expected to happen

See a different wording, like: 

Sign up to view/follow {site-title}.

### What actually happened

Seeing "Sign up to start editing {site-title}" as the screen title

### Impact

All

### Available workarounds?

No but the platform is still usable

### Logs or notes

8275107-zd-a8c

**Comment by Aurorum** [comment]
PR for this issue in #95172. :)

**Comment by inaikem** [comment]
Yep. Thanks for staying on top of your notifications! This was 100% unintentional. The board I just added this to had a default automation to mark issues with the default label as closed. Ouch. Fixing that for ~50 issues as we speak.

**Comment by ironprogrammer** [comment]
The PR @Aurorum submitted, https://github.com/Automattic/wp-calypso/pull/95172, is looking good. Once rebased it should be good to go IMHO.
